gapii/client/capture.go

```diff
@@ -142,7 +142,9 @@ func handleCommError(ctx context.Context, commErr error, anyDataReceived bool) (
 	case errors.Cause(commErr) == io.EOF:
 		log.E(ctx, "unexpected end of stream")
 		abort = true
-		err = commErr
+		// Most of the time, this error happens when the app crashed: rather
+		// than reporting just "EOF", hint that this was probably a crash.
+		err = log.Err(ctx, commErr, "It looks like the app terminated (crashed?) before we could finish the capture")
```

Comment:
Remove the "it looks like" qualifier. I would also prefer a more neutral tone. Maybe "The application exited during tracing"

Reply:
OK, I went with `The application exited during the capture` as most user-facing text seems to use "capture" rather than "trace".